src/main/java/seedu/address/storage/JsonAdaptedCustomer.java

```diff
@@ -81,9 +81,9 @@ public JsonAdaptedCustomer(Customer source) {
     }
 
     /**
-     * Converts this Jackson-friendly adapted person object into the model's {@code Customer} object.
+     * Converts this Jackson-friendly adapted customer object into the model's {@code Customer} object.
      *
-     * @throws IllegalValueException if there were any data constraints violated in the adapted person.
+     * @throws IllegalValueException if there were any data constraints violated in the adapted customer.
```

Comment:
While good to see, is it best practice to have a feature PR and a bug-fix PR mixed together? Worried we might be penalized for bad PR practice

Reply:
Yea, should  e no issue, very very minor edit of the naming of the java doc commetns, not like adding test code